run-tests.pl

```diff
@@ -54,6 +54,7 @@
 binmode(STDOUT, ":utf8");
 
 our $BIND_HOST = "localhost";
+our $ID_SERVER_PORT = 34586;
```

Comment:
Why do we now need to set an explicit port ooi?

Reply:
So that it can be added to trusted servers in homeserver configuration. I suppose that each time `id_server_fixture` is called random port is assigned.

Reply:
Oh yes, I'm blind. We don't need it for Synapse because there's an option to trust all ID servers, right?

Reply:
It is quite messy in Synapse. For example in `/register/email/requestToken` `id_server` field [will be just ignored](https://github.com/matrix-org/synapse/blob/develop/synapse/rest/client/v2_alpha/register.py#L150). I could not figure out how does it work in user-interactive authentication. I didn't either found option to trust all ID servers in Synapse.